src/components/MainPage/HeroSlider/index.tsx

```diff
@@ -60,48 +61,53 @@ const HeroSlider = ({
     speed: 500,

```

Comment:
Краще винести в окремий файл налаштувань
```suggestion
    speed: 500,

```